Userland/Libraries/LibWeb/CSS/GridTrackSize.cpp

```diff
@@ -68,15 +69,15 @@ GridMinMax::GridMinMax(GridSize min_grid_size, GridSize max_grid_size)
 {
 }
 
-DeprecatedString GridMinMax::to_deprecated_string() const
+ErrorOr<String> GridMinMax::to_string() const
 {
     StringBuilder builder;
     builder.append("minmax("sv);
-    builder.appendff("{}", m_min_grid_size.to_deprecated_string());
+    builder.appendff("{}", m_min_grid_size.to_string());
```

Comment:
Be sure to wrap the `.to_string()` s in places like this (that return ErrorOr<>) in `TRY()` to propagate any errors (and also because right now the formatting will be wrong too).
```suggestion
    builder.appendff("{}", TRY(m_min_grid_size.to_string()));
```

Reply:
Ok done! In the `visit` functions this doesn't seem possible. For example in StyleValue.cpp ln 631 (`ErrorOr<String> CalculatedStyleValue::CalcNumberValue::to_string() const`).

The rest is updated though. Thanks again for the help on discord.